src/bank/strict_bank.cairo

```diff
@@ -172,7 +172,11 @@ mod StrictBank {
                 .try_into()
                 .unwrap();
             self.token_balances.write(token, next_balance);
-            next_balance - prev_balance
+            if next_balance >= prev_balance {
+                next_balance - prev_balance
+            } else {
+                0
+            }
```

Comment:
I think we should leave `next_balance - prev_balance` or panic instead of returning 0

Reply:
Updated!